crate_universe/defs.bzl

```diff
@@ -169,6 +168,9 @@ Environment Variables:
             doc = "A template for where to download crates from for the default crate registry. This must contain `{version}` and `{crate}` templates.",
             default = DEFAULT_CRATE_REGISTRY_TEMPLATE,
         ),
+        "iso_date": attr.string(
+            doc = "The iso_date of cargo binary the resolver should use. Note `version` must be `beta` or `nightly`",
```

Comment:
```suggestion
            doc = "The iso_date of cargo binary the resolver should use. Note: This can only be set if `version` is `beta` or `nightly`",
```